docs/Documentation.md

```diff
@@ -0,0 +1,128 @@
+# What's New
+* vm-manager support multiple display on Civ
+* Vm-manager monitors the state of the Civ guest to ensure that the app runs when civ is ready.
+
+# Getting Started
+
+## Prerequisites
+
+1. The host device that launches the virtual machine requires Ubuntu 20.04.
+
+2. Create required directories under $HOME/
+    ```sh
+    mkdir ~/11396
+    mkdir ~/11396/aaf
+    mkdir ~/11396/vtpm0
+    ```
+
+## Run Celadon in a virtual machine
+
+1. Download [caas-releasefiles-userdebug.tar.gz](https://mcg-depot.intel.com/artifactory/cactus-absp-jf/celadon-engineering/11396/caas/userdebug/caas-releasefiles-userdebug.tar.gz) in ``$HOME/11396/`` .
```

Comment:
Do not publish internal link. Use https://github.com/projectceladon/celadon-binary/ instead.